manifests/XMind/XMind 2020/10.2.1.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
```suggestion

```
When I copy paste this manifest, this part contains "TAB".  Either delete tab or delete whole line.

```
    Why does YAML forbid tabs?

    Tabs have been outlawed since they are treated differently by different editors and tools.
    And since indentation is so critical to proper interpretation of YAML, this issue is just too
    tricky to even attempt.  Indeed Guido van Rossum of Python has acknowledged that allowing TABs
    in Python source is a headache for many people and that were he to design Python again, he
    would forbid them.
```